frontend/src/components/common/fields/username-label-field.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * SPDX-FileCopyrightText: 2022 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import { UsernameField } from './username-field'
+import type { UsernameFieldProps } from './username-field'
+import React from 'react'
+import { Form } from 'react-bootstrap'
+import { Trans } from 'react-i18next'
+
+/**
+ * Wraps and contains label and info for UsernameField
+ *
+ * @param onChange Callback that is called when the entered username changes.
+ * @param value The currently entered username.
+ * @param isValid  Is a valid field or not
+ * @param isInvalid Adds error style to label
+ */
+export const UsernameLabelField = (props: UsernameFieldProps) => {
```

Comment:
Please add a type definition.

```suggestion
export const UsernameLabelField: React.FC<UsernameFieldProps> = (props: UsernameFieldProps) => {
```

Reply:
Thanks @ErikMichelson, done